internal/agent/agent.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/flightctl/flightctl/internal/agent/device/status"
 	"github.com/flightctl/flightctl/internal/agent/device/systemd"
 	"github.com/flightctl/flightctl/internal/agent/shutdown"
+	config2 "github.com/flightctl/flightctl/internal/config"
```

Comment:
config2 isn't a great name.  Either rename this one or the first one.

Reply:
renamed the first one